src/main.rs

```diff
@@ -12,6 +12,9 @@ struct Opts {
     /// Instead of reading from STDIN, read the configuration to be applied from the given file.
     #[clap(short, long)]
     file: Option<String>,
+    /// Select netavark's firewall driver
+    #[clap(short='i', long, env="NETAVARK_FW")]
```

Comment:
just remove the short option, we do not use the cli directly so it serves little to no purpose anyway.

Reply:
> just remove the short option, we do not use the cli directly so it serves little to no purpose anyway.

Currently netavark has no way of automatically completing parameters, short parameters may be useful in the future.

Reply:
Netavark is not intended to be ever executed manually on a CLI, I rather just have the long form. Using `-i` just fells random, who is going to remember what it means?

Reply:
> Netavark is not intended to be ever executed manually on a CLI, I rather just have the long form. Using `-i` just fells random, who is going to remember what it means?

I decided to give up until I found the right short parameters.